README.md

```diff
@@ -180,6 +180,71 @@ spec:
    master: eno3
 ```
 
+### User Defined Injections
+
+User Defined injections allows user to define additional injections (besides what's supported in NRI, such as ResourceName, Downward API volumes etc) in Kubernetes ConfigMap and request additional injection for individual pod based on pod label. Currently customized injection only support injecting pod annotations.
```

Comment:
IMO if user defined injections is used, we should use it all the way.
in the content of this section, examples and code. The word "customized" should be replaced with "user defined" so we are consistent throughout.